attributes/default.rb

```diff
@@ -219,43 +221,25 @@
                                                        )
                                                      else
                                                        value_for_platform(
-                                                         'default' => "el/7/x86_64",
-                                                         'ubuntu' => {
-                                                           '20.04' => "ubuntu/20.04/x86_64",
-                                                           '18.04' => "ubuntu/18.04/x86_64",
-                                                         }
+                                                         'default' => "el/7/x86_64"
                                                        )
                                                      end
 default['cluster']['mysql']['package']['file-name'] = "mysql-community-client-#{node['cluster']['mysql']['package']['version']}.tar.gz"
 default['cluster']['mysql']['package']['archive'] = "#{node['cluster']['mysql']['package']['root']}/#{node['cluster']['mysql']['package']['platform']}/#{node['cluster']['mysql']['package']['file-name']}"
 default['cluster']['mysql']['package']['source'] = "#{node['cluster']['artifacts_s3_url']}/source/mysql-#{node['cluster']['mysql']['package']['source-version']}.tar.gz"
 
 # MySQL Validation
-if arm_instance?
-  default['cluster']['mysql']['repository']['packages'] = value_for_platform(
-    'default' => %w(mysql-community-devel mysql-community-libs mysql-community-common mysql-community-client-plugins mysql-community-libs-compat),
-    'ubuntu' => {
-      'default' => %w(libmysqlclient-dev libmysqlclient21),
-      '18.04' =>  %w(libmysqlclient-dev libmysqlclient20),
-    }
-  )
-  default['cluster']['mysql']['repository']['expected']['version'] = value_for_platform(
-    'default' => "8.0.31"
-    # Ubuntu 18.04/20.04 ARM: MySQL packages are installed from OS repo and we do not assert on a specific version.
-  )
-else
-  default['cluster']['mysql']['repository']['packages'] = value_for_platform(
-    'default' => %w(mysql-community-devel mysql-community-libs mysql-community-common mysql-community-client-plugins mysql-community-libs-compat),
-    'ubuntu' => { 'default' => %w(libmysqlclient-dev libmysqlclient21 mysql-common mysql-community-client-plugins) }
-  )
-  default['cluster']['mysql']['repository']['expected']['version'] = value_for_platform(
-    'default' => "8.0.31",
-    'ubuntu' => {
-      '20.04' => "8.0.31-1ubuntu20.04",
-      '18.04' => "8.0.31-1ubuntu18.04",
-    }
-  )
-end
+default['cluster']['mysql']['repository']['packages'] = value_for_platform(
+  'default' => %w(mysql-community-devel mysql-community-libs mysql-community-common mysql-community-client-plugins mysql-community-libs-compat),
+  'ubuntu' => {
```

Comment:
You moved the `arm` case as default. It means for ubuntu x86 we had:
```
    'ubuntu' => { 'default' => %w(libmysqlclient-dev libmysqlclient21 mysql-common mysql-community-client-plugins) }
``` 
and now we have:
``` 
    'ubuntu' => {
      'default' => %w(libmysqlclient-dev libmysqlclient21),
      '18.04' =>  %w(libmysqlclient-dev libmysqlclient20),
    }
```

Is this correct?

Reply:
It should be correct. That wasn't the "arm" case, but rather the case of using the packages from the OS repos, which was used for arm. Now all Ubuntu builds default to that behavior.